src/main/java/org/dita/dost/module/GenMapAndTopicListModule.java

```diff
@@ -748,7 +753,7 @@ private void outputResult() throws DITAOTException {
             getOrCreateFileInfo(fileinfos, file.filename).format = file.format;
         }
         for (final URI file: flagImageSet) {
-            final FileInfo f = getOrCreateFileInfo(fileinfos, file);
+  		  final FileInfo f = getOrCreateFlagImageFileInfo(fileinfos, file, flagImagesMap.get(file));
```

Comment:
Fix indentation. Are you using 4 spaces?

Reply:
(sigh) might be, unintentionally - I just set up Eclipse on my new system last week and hadn't changed settings to match the old system.